CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Added
 
+## Changed
+
+# v0.16.0
+
+## Added
```

Comment:
The following need to be part of the `Added` section instead:
- Add support for coalesced MMIO (KVM_CAP_COALESCED_MMIO) from PR #244
- Add support for userspace NMI injection (KVM_NMI / KVM_CAP_USER_NMI) from #246
- Add support for userspace MSR handling #241 
- Add support for KVM_CAP_X86_SMM / KVM_SMI #242 

Can you please move them from `Changed`?